site/en/docs/android/custom-tabs/translate-language/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: "layouts/doc-post.njk"
+title: Translate your content
+seoTitle: "Guide: Localize Content in a Custom Tab"
+date: 2023-04-21
+description: How to translate content in a Custom Tab.
+authors:
+  - alexmkatz
+---
+
+When a user opens a Custom Tab, the content in that tab is usually translated for their device locale. This guide explains how to implement Custom Tab translation for a locale of your choice instead.
```

Comment:
> When a user opens a Custom Tab, the content in that tab is usually translated for their device locale.

As discussed offline, there should not be automatic translation happening here. I think we should say here something similar to what the setTranslateLocale() JavaDoc says[1]. Maybe it could be something similar to as this is basically what is happening:

"This guide explains how to specify the the target locale the Custom Tabs Translate UI should be triggered with."

What happens when this intent extra is set is that if the user opens a CCT, goes to the 3-dot menu and taps "Translate..." they will get the page translated to the language they set up in the intent extra. However, manual interaction is still required by the user as in the workflow explained above.

[1] https://developer.android.com/reference/androidx/browser/customtabs/CustomTabsIntent.Builder#setTranslateLocale(java.util.Locale)

Reply:
Updated.